### ISSUE TITLE
Feature request: Signature expiration

### ISSUE THREAD

**Issue by tylergets** [cshum/imagor#244]
Hey there, big thanks again for this project.

One use case my company is exploring is including an expiration time within the imagor signature, making links invalid after a certain timeframe.

We are currently expirementing doing this with cloudfront however having it build directly into imagor would be ideal as it would avoid two signatures. 

I would be happy to discuss how we could support the development of this!

**Comment by cshum** [comment]
What comes up in my mind is an `expire(timestamp)` filter where `timestamp` is the unix milliseconds timestamp of expiration time.

The expire filter would be evaluated as part of the URL signature.

**Comment by tylergets** [comment]
Yeah that makes sense to me and fits in nicely; only thing I see is that expire time should not be included when saving the image to storage

**Comment by cshum** [comment]
You may test the docker build at master branch:
https://github.com/cshum/imagor/pkgs/container/imagor/50398420?tag=master
```
ghcr.io/cshum/imagor@sha256:24bea2d2397725de24da17c6f522b0fed382d937e3a3ab0de74f1babe6dc8049
```

The `expire(timestamp)` filter functionality, and it should be excluded from result storage path

**Comment by tylergets** [comment]
Works great in our testing, thank you!

**Comment by cshum** [comment]
available in v1.3.0

**Comment by rela589n** [comment]
Hi there!

Thanks for all your effort maintaining this amazing project!

I think it would be a good thing to write this statement in README:

> The expire(timestamp) filter is excluded from result storage path
